aikau/.jshintrc

```diff
@@ -30,6 +30,7 @@
       "module": false,
       "model": false,
       "appContext": false,
+      "JSON": false,
```

Comment:
Even though JSON parse/stringify is supported by even IE8, it's not part of ES3, so previously JSHint would warn about using it. This explicitly declares it as a valid global object (false means not modifiable).